_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure istio ingress with an external load balancer NLB on AWS
+publish_date: April 08, 2018
+subtitle: Ingress AWS NLB
```

Comment:
Why not spell out NLB?  Just a question, not a suggestion for change.

Reply:
Indeed